weave-js/src/components/Panel2/PanelTable.styles.ts

```diff
@@ -99,18 +98,23 @@ FilterIcon.displayName = 'S.FilterIcon';
 
 export const ColumnAction = styled.div<{isHovered?: boolean}>`
   cursor: pointer;
+  height: 100%;
+  background-color: transparent;
+`;
+ColumnAction.displayName = 'S.ColumnAction';
+
+export const ColumnActionContainer = styled.div<{isHovered?: boolean}>`
+  display: flex;
   padding: 5px 0px 0px 0px;
+  font-size: 20px;
```

Comment:
Do we have a common font-size variable we can access here instead of a hard-coded px? non-blocking as this was here before

Reply:
As far as I know, I don't think so. This `font-size` property is used only to size the indicator and ellipses icons